en/setup/dev_env_windows_vm.md

```diff
@@ -0,0 +1,29 @@
+# Windows Virtual Machine-Hosted Toolchain
+
+Windows developers can run the PX4 toolchain in a virtual machine with linux as the guest operating system. After setting up the virtual machine the installation and setup of PX4 within the VM is exactly the same as on a native Linux computer.
+
+> **Tip** Allocate as many CPU cores and memory resources to the VM as possible.
+
+While using a VM is a very easy way to set up and test an environment for building firmware, users should be aware:
+1. Firmware building will be slower than native building on Linux.
+1. The JMAVSim frame rate be much slower than on native Linux. In some cases the vehicle may crash due to issues related to insufficient VM resources.
+1. Gazebo and ROS can be installed, but are unusably slow.
+
+## Instructions
+
+There are multiple ways to setup a VM which is capable of executing the PX4 environment on your system. This guide walks you through a VMWare setup which showed usable performance for basic usage not including ROS usage and high resolution heavy graphics applications.
```

Comment:
New step by step guide for VMWare Player (which has to my knowledge best performance and is free).
Still rough, more details and screenshots to come...